website/docs/guides/version-3-upgrade.html.md

```diff
@@ -25,6 +25,7 @@ Upgrade topics:
 - [Data Source: aws_lambda_invocation](#data-source-aws_lambda_invocation)
 - [Data Source: aws_route53_resolver_rule](#data-source-aws_route53_resolver_rule)
 - [Data Source: aws_route53_zone](#data-source-aws_route53_zone)
+- [Data Source: aws_launch_template](#data-source-aws_launch_template)
```

Comment:
Hi @arielpeltz 👋 Do you mind moving this line below `aws_lambda` to maintain the list in alphabetical order?